### PR TITLE
`Improvement` indv. poolpair page

### PR DESCRIPTION
**Pull request by siradji**
#### What kind of PR is this?:

/kind feature

#### Which issue(s) does this PR fixes?:

Fixes #

#### Sample Links & Screenshots:

Link: 
<details>
<summary>Desktop Screenshot</summary>

</details>
<details>
<summary>Mobile Screenshot</summary>

</details>

#### Developer Checklist:

- [ ] Read your code changes at least once
- [ ] Tested on multiple web browsers
- [ ] Tested responsiveness (e.g, iPhone, iPad, Desktop)
- [ ] No console errors
- [ ] Unit tests*
- [ ] Added e2e tests*

### CODE REVIEW

**Comment by siradji** [comment]
@joeldavidw, is the bug fixed in [0.31.2](https://github.com/DeFiCh/whale/releases/tag/v0.31.2)?

**Comment by nichellekoh** [comment]
> @joeldavidw, is the bug fixed in [0.31.2](https://github.com/DeFiCh/whale/releases/tag/v0.31.2)?

Yup, the fix is pushed in that version. Let's verify if it works

**Comment by siradji** [comment]
> > @joeldavidw, is the bug fixed in [0.31.2](https://github.com/DeFiCh/whale/releases/tag/v0.31.2)?
> 
> Yup, the fix is pushed in that version. Let's verify if it works once it gets deployed tomorrow

okay!!

File: src/pages/dex/[poolpairId]/_components/SwapCards.tsx [hunk 1]
```diff
@@ -23,32 +21,22 @@ export function SwapCards ({ swaps }: { swaps: PoolSwapData[]}): JSX.Element {
...
-    }
-  }, [swap.block.medianTime])
-
+  const age = useAge(swap.block.medianTime)
   return (
     <CardList.Card testId='SwapCard'>
       <CardList.Header>
-        <SwapTokens from={swap.from?.symbol} to={swap.to?.symbol} />
+        <TxIdLink txid={swap.txid} testId='CardList.Header.TxId'>
+          <TextTruncate text={swap.txid} className='w-44' />
```
Comment (nichellekoh) [inline src/pages/dex/[poolpairId]/_components/SwapCards.tsx:29]: Are we displaying the txid as link instead of the displaySymbols?

**Comment by siradji** [comment]
@joeldavidw , the new fix is working fine. will need your review on this

File: src/pages/dex/[poolpairId]/_components/SwapTable.tsx [hunk 2]
```diff
@@ -1,21 +1,23 @@
...
     <div data-testid='SwapTable'>
       <OverflowTable className='mt-4'>
         <OverflowTable.Header>
-          <OverflowTable.Head title='Tx ID' />
+          <OverflowTable.Head title='Transaction ID' />
           <OverflowTable.Head title='Age' />
-          <OverflowTable.Head title='Amount' />
+          <OverflowTable.Head title='Type' />
+          <OverflowTable.Head title='Input Amount' alignRight />
+          <OverflowTable.Head title='Output Amount' alignRight />
```
Comment (nichellekoh) [inline src/pages/dex/[poolpairId]/_components/SwapTable.tsx:20]: Do we only align right for these 2 columns?

**Review by nichellekoh — APPROVED** [review]

**Comment by nichellekoh** [comment]
For issue https://github.com/DeFiCh/scan/issues/1049

**Pull request merged**